src/Grid.php

```diff
@@ -513,6 +513,32 @@ public function addModalAction($button, $title, \Closure $callback, $args = [])
         return $this->getActionButtons()->addModal($button, $title, $callback, $this, $args);
     }
 
+    /**
+     * Similar to addModalAction but apply to a multiple recors selection and display in menu.
+     * When menu item is clicked, modal is displayed with the $title and $callback is executed through
+     * VirtualPage.
+     *
+     * @param string|array|MenuItem            $item
+     * @param \Closure(View, array|null): void $callback
```

Comment:
why to pass `null` when `$t->stickyGet($this->name)` is empty?

Reply:
fix this, add a Behat test and I will do a final review

TODO for myself: impl. to managed ID typecasting here and also in https://github.com/atk4/ui/blob/4.0.0/src/Table/Column/ActionButtons.php#L99

Reply:
Replaced null by empty array and added behat test